src/main/java/frc/robot/Match.java

```diff
@@ -27,6 +31,21 @@ public void robotInit()
   {
     // instantiate the robot
     robot = new Robot();
+    auto = new SequentialCommandGroup(
+
+      new AutoDrive(robot, -.5, 0, 2), // drive backwards
+      new AutoShoot(robot, 3, .4) //shoot
+
+      // Robot on right tarmac
+
+      //new AutoDrive(robot, 0, -.5, 1.2), //turn left
+      //new AutoDrive(robot, .5, 0, 1.5) //drive away
+
+      //Robot on left tarmac
+      
+      //new AutoDrive(robot, 0, .5, 1.2), //turn right
+      //new AutoDrive(robot, .7, 0, 1.2) //drive away
+    );
```

Comment:
You should make a new file for this command group with a class that inherits from `SequentialCommandGroup`.  Then just import that one command and start it here.  Will keep things more organized if you add more auto command groups later on.